apps/app/src/gql/fragment-masking.ts

```diff
@@ -0,0 +1,66 @@
+import { ResultOf, DocumentTypeDecoration, TypedDocumentNode } from '@graphql-typed-document-node/core';
```

Comment:
wdyt about not committing this folder?

https://github.com/dotansimha/graphql-code-generator/discussions/4253

Reply:
I went back and forth. It'd require adding a build step and causes some issues when people check the code out. But yeah we should get rid of it soon-ish